src/platform/user/authentication/utilities.js

```diff
@@ -105,16 +113,16 @@ function redirect(redirectUrl, clickedEvent) {
 }
 
 export function login(policy, version = 'v0', application = null, to = null) {
-  return redirect(
-    loginUrl(policy, version, application, to),
-    'login-link-clicked-modal',
-  );
+  const url = loginUrl(policy, version, application, to);
+  setForceAuth();
+  return redirect(url, 'login-link-clicked-modal');
 }
 
 export function autoLogin() {
   const url = sessionTypeUrl('idme', 'v1', undefined, undefined, {
     inbound: 'true',
```

Comment:
what's the logic that `vets-api` is applying based on this param? anything that would affect subsequent client-side interactions?

Reply:
if login fails for any reason, and the `?inbound=true` query parameter was specified, then the response will contain `?auth=force-needed` rather than `?auth=fail`.  On the FE side we just ignore the former and return the user to where they left off rather than displaying the failure page.